Update macOS image on Azure pipelines

The macOSX 10.13 VM are being retired.
```
##[error]This job was temporarily blocked because it uses a Microsoft hosted agent (MacOS-10.13) that will be permanently removed on March 23rd, 2020.  See https://aka.ms/blocked-hosted-agent for more information.
,##[error]The remote provider was unable to process the request.
```
See https://devblogs.microsoft.com/devops/removing-older-images-in-azure-pipelines-hosted-pools/

We need to replace the 10.13 image with the 10.14 in our azure pipelines

--- a/AUTHORS.rst
+++ b/AUTHORS.rst
@@ -18,3 +18,4 @@ The following organizations or individuals have contributed to ScanCode:
 - Mrinal Paliwal @mriiinal
 - Mankaran Singh @MankaranSingh
 - Ravi Jain @JRavi2
+- Tushar Goel @TG1999

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -4,6 +4,8 @@ Changelog
 Outputs:
  - The "headers" attribute in JSON outputs now contains a 'duration' field. #1942
 
+OS support:
+ - Add 10.15 macOS tests to azure-pipelines
 
 v3.0.2 (2019-02-15)
 -------------------

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -83,8 +83,8 @@ jobs:
 
     - template: etc/ci/azure-mac.yml
       parameters:
-          job_name: macos1013_py27
-          image_name: macos-10.13
+          job_name: macos1014_py27
+          image_name: macos-10.14
           python_versions: ['2.7']
           test_suites:
               all: bin/py.test -n 2 -vvs --reruns=3 --ignore=tests/scancode
@@ -92,12 +92,12 @@ jobs:
 
     - template: etc/ci/azure-mac.yml
       parameters:
-          job_name: macos1014_py27
-          image_name: macos-10.14
+          job_name: macos1015_py27
+          image_name: macos-10.15
           python_versions: ['2.7']
           test_suites:
               all: bin/py.test -n 2 -vvs --reruns=3 --ignore=tests/scancode
-              scancode: bin/py.test -vvs --reruns=3 tests/scancode
+              scancode: bin/py.test -vvs --reruns=3 tests/scancode          
 
 
 ################################################################################
@@ -236,19 +236,19 @@ jobs:
 
     - template: etc/ci/azure-mac.yml
       parameters:
-          job_name: macos_1013_py36
-          image_name: macos-10.13
+          job_name: macos1014_py36
+          image_name: macos-10.14
           python_versions: ['3.6']
           test_suites:
               all: bin/py.test -n 2 -vvs --reruns=3
 
     - template: etc/ci/azure-mac.yml
       parameters:
-          job_name: macos1014_py36
-          image_name: macos-10.14
+          job_name: macos1015_py36
+          image_name: macos-10.15
           python_versions: ['3.6']
           test_suites:
-              all: bin/py.test -n 2 -vvs --reruns=3
+              all: bin/py.test -n 2 -vvs --reruns=3          
 
     - template: etc/ci/azure-win.yml
       parameters:

--- a/docs/source/getting-started/whats-new.rst
+++ b/docs/source/getting-started/whats-new.rst
@@ -49,7 +49,7 @@ deprecated hereafter. The new documentation has improved support in terms of:
 #. Consistent with the Latest Version
 #. Command Line Interface Reference
 #. Getting Started Support for Newcomers
-#. Restructured to fit into the recommended `4 Category Doc Format <https://www.divio.com/blog/documentation/>`_
+#. Restructured to fit into the recommended `4 Category Doc Format <https://docs.divio.com/en/latest/>`_
 #. Updated Tutorials/How To's
 #. Updated Plugin Support
 

--- a/docs/source/license.rst
+++ b/docs/source/license.rst
@@ -3,4 +3,4 @@ License
 
 ScanCode Toolkit includes documents that are dedicated to the Public Domain using the
 Creative Commons CC0 1.0 Universal (CC0 1.0) Public Domain Dedication:
-http://creativecommons.org/publicdomain/zero/1.0/
+https://creativecommons.org/publicdomain/zero/1.0/

--- a/docs/source/misc/faq.rst
+++ b/docs/source/misc/faq.rst
@@ -37,7 +37,7 @@ crafted regex):
   a grammar; it has a few thousand tests.
 - licenses and copyrights are detected in texts and binaries
 
-Licensecheck (available here for reference: /https://metacpan.org/release/App-Licensecheck )
+Licensecheck (available here for reference: /https://metacpan.org/pod/App::Licensecheck )
 is a Perl script using hand-crafted regex patterns to find typical copyright statements and
 about 50 common licenses. There are about 50 license detection tests.
 

